edgecompute/examples/traffic-routing/url-shortner/main.js

```diff
@@ -0,0 +1,147 @@
+/*
+(c) Copyright 2022 Akamai Technologies, Inc. Licensed under Apache 2 license.
+
+Version: 0.5
+Purpose:  Using EdgeWorkers and EdgeKV, manage a list of shortened URLs and serve redirects for them
+*/
+
+import URLSearchParams from 'url-search-params';  
+import { logger } from 'log';
+import { EdgeKV } from './edgekv.js';
+import { createResponse } from 'create-response';
+
+export async function responseProvider (request) {
+  // Set Up EdgeKV
+  const edgeKv = new EdgeKV({namespace: "url-shortener", group: "links"});
+
+  // Request path
+  const path = request.path;
+
+  // If the request path contains /r/ then we are serving a redirect
+  if (path.split('/')[1] === 'r') {
+    let key = path.split('/')[2];

```

Comment:
I suggest you sanitize the key here since it is passed in by a user without any further checks, so could present an opportunity for exploits.